polaris-react/src/components/BulkActions/BulkActions.scss

```diff
@@ -7,76 +7,55 @@ $bulk-actions-button-stacking-order: (
 
 .Group {
   @include text-style-input;
-  width: 100%;
-  display: none;
+  display: flex;
   align-items: center;
   flex-wrap: wrap;
   opacity: 0;
+  width: 100%;
+  justify-content: center;
+  transition: var(--p-duration-250) var(--p-ease-in);
+  transition-property: transform, opacity;
+  transform: translateY(120px);
+  padding: 0 var(--p-space-4);
+
+  &.Group-not-sticky {
+    transform: none;
+    opacity: 1;
+  }
 
-  &.Group-measuring {
-    transition: none;
-    display: flex;
+  &.Group-entering,
+  &.Group-exiting {
     opacity: 0;
+    display: flex;
   }
-}
-
-.Group-entering,
-.Group-exiting {
-  opacity: 0;
-  display: flex;
-}
-
-.Group-entered {
-  opacity: 1;
-  display: flex;
-}
 
-.Group-exited {
-  opacity: 0;
-  display: none;
-}
+  &.Group-entered {
+    opacity: 1;
+    display: flex;
+    transform: translateY(0);
+  }
 
-.Group-smallScreen {
-  @media #{$p-breakpoints-sm-up} {
+  &.Group-exited {
+    opacity: 0;
     display: none;
   }
-}
 
-.Group-largeScreen {
-  display: none;
-
-  @media #{$p-breakpoints-sm-up} {
+  &.Group-measuring {
+    transition: none;
     display: flex;
-
-    &.Group-exiting {
-      transition: none;
-    }
-
-    &.Group-exited {
-      opacity: 0;
-      display: none;
-    }
+    opacity: 0;
   }
 }
 
 .ButtonGroupWrapper {
-  width: 100%;
+  width: auto;
+  justify-content: flex-start;
+  padding: var(--p-space-4);
+  background: var(--p-surface);
+  border-radius: var(--p-border-radius-2);
+  box-shadow: var(--p-shadow-popover);
```

Comment:
^ (similar comment) - issues recently on safari, however, being only one element I suspect it won't be an issue